akka-remote/src/main/scala/akka/remote/artery/compress/CompressionTable.scala

```diff
@@ -94,10 +95,15 @@ private[remote] object CompressionTable {
   }
   def compareBy2ndValue[T]: Comparator[Tuple2[T, Int]] = CompareBy2ndValue.asInstanceOf[Comparator[(T, Int)]]
 
-  def empty[T] = new CompressionTable[T](0, 0, new Object2IntHashMap[T](NotCompressedId))
+  private def newObject2IntHashMap[T]: Object2IntHashMap[T] = {
+    // import to use shouldAvoidAllocation = false because of concurrent access of dictionary
+    new Object2IntHashMap[T](256, Hashing.DEFAULT_LOAD_FACTOR, NotCompressedId, false)
```

Comment:
i guess 256 is more useful than 8 as a min cap

Reply:
our default in reference.conf is 256 so I thought it would make sense to increase here

Reply:
bug fix is: `shouldAvoidAllocation = false`

Reply:
but maybe better to keep the initialCapacity small for the empty case, let me adjust

Reply:
improved that: https://github.com/akka/akka/pull/28363/commits/5f8a424308c11c7b56098750c238bea1a874a449

Reply:
:+1: